packages/nocms/src/lib/plugins.js

```diff
@@ -3,15 +3,15 @@ import path from 'path';
 
 export async function loadPlugins(pluginActivationContext) {
 	return Promise.all([
-		findFilesAsync('node_modules/nocms-plugin-*/', {})
+		findFilesAsync(`${process.cwd()}/node_modules/nocms-plugin-*/`, {})
 			.then(pluginDirectories => pluginDirectories
 				.map(pluginsDirectory => path.basename(pluginsDirectory))),
 		findFilesAsync('plugins/nocms-plugin-*/', { cwd: __dirname })
 			.then(pluginDirectories => pluginDirectories
 				.map(pluginsDirectory => './' + pluginsDirectory))
 	])
 		.then(modules => [].concat.apply([], modules))
-		.then(modules => modules.map(moduleName => require(moduleName)))
+		.then(modules => modules.map(moduleName => require(path.join(process.cwd(), 'node_modules', moduleName))))
```

Comment:
Revert as well I suppose

Reply:
No, this should be true for all runs, since the process.cwd() is where the nocms command is run, for instance standalone on the command line or initiated through the npm scripts section

Reply:
Please use path functions to join paths